libraries/standard/mqtt/src/iot_mqtt_validate.c

```diff
@@ -34,6 +34,21 @@
 /* MQTT internal include. */
 #include "private/iot_mqtt_internal.h"
 
+/**
+ * @brief Check that an #IotMqttPublishInfo_t is valid
+ *
+ * @param[in] awsIotMqttMode Specifies if this PUBLISH packet is being sent to
+ * an AWS IoT MQTT server.
+ * @param[in] pPublishTypeDescription String describing the publish type.
```

Comment:
Missing documentation for `maximumPayloadLength`

Reply:
Fixed

Reply:
Nit: period at end of comment